examples/workflows/custom_base_node/README.md

```diff
@@ -0,0 +1,32 @@
+# Custom Base Node
```

Comment:
@awlevin Could I get some help with README's for the rest of the examples we have set up?